draft-quic-atsss-reqs.mkd

```diff
@@ -264,8 +264,8 @@ The Proxy enables the UE to use a transport protocol that supports
 the different access networks even if the server does not support it (i.e., is not multipath-capable
 at transport layer).
 
-The 5G control plane provides the UE and the Proxy with ATSSS rules as discussed in {{sec-rules}}.
-
+The 5G control plane provides the UE and the Proxy with rules that specify
```

Comment:
Mirja, any reason to move this text back here?

Reply:
Ups that was not the intention. Just wanted to edit the wording.